README.rst

```diff
@@ -0,0 +1,158 @@
+
+For more information on deploying DeepCell in the cloud refer to the `DeepCell Kiosk documentation <https://deepcell-kiosk.readthedocs.io>`_.
+
+Examples
+--------
+
+.. list-table::
+
+    * - Raw Image
+      - Segmented and Tracked
+    * - .. image:: docs/images/raw.gif
+      - .. image:: docs/images/tracked.gif
+
+Getting Started
+---------------
+
+The fastest way to get started with DeepCell is to run the latest docker image:
+
+.. code-block:: bash
+
+    nvidia-docker run -it --rm -p 8888:8888 vanvalenlab/deepcell-tf:latest
+
+This will start a jupyter session, with several example notebooks detailing various training methods:
+
+For examples of how to use the deepcell library, check out the following notebooks:
+* `Training a segmentation model <https://deepcell.readthedocs.io/en/master/Training-Segmentation.html>`_
+* `Training a tracking model <https://deepcell.readthedocs.io/en/master/Training-Tracking.html>`_
+
+``deepcell.applications`` and ``deepcell.datasets``
+---------------------------------------------------------
+Together |deepcell.datasets| and |deepcell.applications|  work together to provide an accessible entrypoint to deep learning for biologists. The datasets module contains a variety of annotated datasets that can be used as training data. Additionally, the applications package initializes a set of models complete with the option to initialize with pre-trained weights.
+
+.. |deepcell.datasets| raw:: html
+
+    <tt><a href="https://deepcell.readthedocs.io/en/master/API/deepcell.datasets.html">deepcell.datasets</a></tt>
+
+.. |deepcell.applications| raw:: html
+
+    <tt><a href="https://deepcell.readthedocs.io/en/mrgn-docs/API/deepcell.applications.html">deepcell.applications</a></tt>
+
+DeepCell for Developers
+-----------------------
+
+DeepCell uses ``nvidia-docker`` and ``tensorflow`` to enable GPU processing. If using GCP, there are `pre-built images <https://console.cloud.google.com/marketplace/details/nvidia-ngc-public/nvidia_gpu_cloud_image>`_ which come with CUDA, docker, and nvidia-docker pre-installed. Otherwise, you will need to install `docker <
+https://docs.docker.com/install/linux/docker-ce/debian/>`_, `nvidia-docker <https://github.com/NVIDIA/nvidia-docker>`_, and `CUDA <https://developer.nvidia.com/cuda-downloads>`_ separately.
+
+Build a local docker container, specifying the tensorflow version with TF_VERSION
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+    git clone https://github.com/vanvalenlab/deepcell-tf.git
+    cd deepcell-tf
+    docker build --build-arg TF_VERSION=1.15.0-gpu -t $USER/deepcell-tf .
+
+
+Run the new docker image
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+    # NV_GPU refers to the specific GPU to run DeepCell on, and is not required
+
+    NV_GPU='0' nvidia-docker run -it \
+    -p 8888:8888 \
+    $USER/deepcell-tf:latest
```

Comment:
Let's use `0.4.0-gpu` instead of `latest` as the tag. We will need to update this with future releases but that's easy.  I think this is clearer, and less likely to have user issues in the future.